contrib/mesos/pkg/offers/offers.go

```diff
@@ -79,16 +81,16 @@ type RegistryConfig struct {
 	DeclineOffer  func(offerId string) <-chan error // tell Mesos that we're declining the offer
 	Compat        func(*mesos.Offer) bool           // returns true if offer is compatible; incompatible offers are declined
 	TTL           time.Duration                     // determines a perishable offer's expiration deadline: now+ttl
-	LingerTTL     time.Duration                     // if zero, offers will not linger in the FIFO past their expiration deadline
+	LingerTTL     time.Duration                     // if zero, offers will not linger in the FIFO past their event time
```

Comment:
"expiration deadline" in the context of offers makes more sense than "event time" -- "event" is too generic here.